src/databricks/labs/ucx/hive_metastore/migrate_tables.py

```diff
@@ -0,0 +1,61 @@
+import logging
+from functools import partial
+
+from databricks.sdk import WorkspaceClient
+
+from databricks.labs.ucx.framework.crawlers import SqlBackend
+from databricks.labs.ucx.framework.parallel import ThreadedExecution
+from databricks.labs.ucx.hive_metastore.tables import TablesCrawler
+
+logger = logging.getLogger(__name__)
+
+
+class TablesMigrate:
```

Comment:
Move this class to the end of tables.py, so everything related to tables is confined in a single  python module